llm_studio/app_utils/sections/chat.py

```diff
@@ -187,6 +197,8 @@ def gpu_is_blocked(q, gpu_id):
     )
     return gpu_blocked
 
+def get_random_prediction_sample(validation_dataframe: pd.DataFrame):
+    return validation_dataframe.iloc[:, 0].sample().item()
```

Comment:
I looked it up. `cfg` does indeed have information about the prompt_column. I found it at `cfg.dataset.prompt_column`. However, it returned me with a tuple as `('instruction', )`. I'm assuming this could be if there are multiple columns to be used in constructing the prompt like in the case of system prompts. For now, I have just used the first column in this tuple, but maybe later on we can add option for multiple prompt columns as well.

So, for now, I used `cfg.dataset.prompt_column[0]`.